pylintrc

```diff
@@ -355,10 +288,9 @@ no-docstring-rgx = __.*__$|test_.+|setUp$|setUpClass$|tearDown$|tearDownClass$|M
 docstring-min-length = 5
 
 [FORMAT]
-max-line-length = 100
+max-line-length = 120
```

Comment:
@samuelallan72 We have the line length being set to 120 here. But this config isn't shared with Black. So black uses the default for `88` for formatting.

Black seems to load the config [only from "pyproject.toml"](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file) at the project level. We should either add that file and set the `line-length` to `120` for consistency or at least pass it as an CLI argument in the Makefile `black -l 120`. But later is sub-optimal.

Can we add a pyproject.toml and put Black configuration there? I think that would actually make this diff a lot smaller than now.

Reply:
@samuelallan72 Thank you for addressing this. I think I clicked Add comment instead of "Review" for this comment and it got published before the rest of the comments. I am sorry if it caused confusion for you.

Reply:
All good :grin: